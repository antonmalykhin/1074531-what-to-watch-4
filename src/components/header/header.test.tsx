import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import history from '../../history';
import Header from './header';



it(`Header render`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <Header classNameModifier={`user-page`}>
          <div />
        </Header>
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
