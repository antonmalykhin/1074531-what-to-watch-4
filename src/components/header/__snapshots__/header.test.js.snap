// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render 1`] = `
Array [
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>,
  <header
    className="page-header movie-card__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div />
  </header>,
]
`;
